docs/getting-started.md

```diff
@@ -90,24 +92,24 @@ example-python: Build ID: 01BSY5R8J45QHG9D3B17PAXMGN
 Now that the application has been deployed, we can connect to our app.
 
 ```shell
-$ draft connect
-Connect to python:8080 on localhost:39551
+$ draft connect -p 8081:8080
```

Comment:
small nit. Since this is a getting started guide, we may not need to put the `-p` flag example here since that is more of an advanced use case and this doc is for beginners. In addition, since the `override-ports` field is shown in the example `draft.toml`, the `-p` flag scenario may be confusing. What do you think?

Reply:
I agree. We should really strive for simplification where possible. `-p` isn't necessary in the base case, so I think it should be omitted from the Getting Started guides.